Topics/Development_Process/Azure_Functions.md

```diff
@@ -0,0 +1,193 @@
+# Introduction to Azure Functions
+
+This tutorial will help you get started with Azure Functions.
+
+## What are Azure Functions
+
+Azure Functions are a serverless computation service offered by Azure. They allow developers to build and deploy event-driven business logic. Benefits include:
+
+- Very easy to set up and deploy (as we will soon see). Also has options to integrate with CI/CD
+- Flexible on the language of choice
+- Responds to different types of triggers; HTTP, time scheduler, Event from Azure CosmosDB, etc.
+- Automatically scales
+- Pay for what you consume, more cost effective than paying for a VM
+
+Some cons:
+
+- Azure Functions on the Consumption plan will have some latency when invoked for the first time in a while ("cold start")
+- You're tied to a specific vendor (i.e. Microsoft)
+- Azure Functions have a set execution time, it will stop running if execution is taking too long
+
+## Azure Setup
+
+Here we will go through everything we need to do in the [Azure Portal](https://portal.azure.com/).
+
+### Account Signup
+
+First, you will need an Azure account. If you don't have one, sign up for [Azure](https://azure.microsoft.com/).
+
+**Disclaimer**: You will need a credit card. If you are new to Azure, you are eligible for a free tier. Moreover, you can link your UofT email to your Azure account to be for further credit. Microsoft will only put a small holding fee to verify your identity.
```

Comment:
_Moreover, you can link your UofT email to your Azure account to be for further credit._

What's the benefit of using the UofT email? I don't totally get it. 

If there is some benefit, try making it more general, since non UofT folks will likely see and use this site. E.g. "You can link your institution email address to your account which will make X easier". 

If there is no benefit, I wouldn't mention UofT then.

Reply:
after re reading.. did you mean that the person gets more free credits? If so, I would rephrase to "Moreover, you can link your university email (if you have one) to your Azure account to get more free credits" or something

Reply:
Thank you for your suggestion, Addison!  Yes, linking your UofT email to your Azure account can grant you Azure credit, which will subsequently be used to pay for Azure services once your free tier is over.

I've gone ahead and updated the file to reflect this.